Automattic/sensei
# Add new purpose step

Fixes #5644

### Changes proposed in this Pull Request

* It updates the purpose steps with the new options and the new style.

### Testing instructions


* Navigate to the Setup Wizard (`/wp-admin/admin.php?page=sensei_setup_wizard`).
* Click on "Get started" to go to the purpose step.
* Select some options (make sure the 2 first ones display an extra text when selected, and the last one displays an input text).
* Click on "Continue".
* Make sure it goes to the usage tracking.
* Use the "Back" from the browser, refresh the page, and make sure your selected options were persisted.


### Screenshot / Video

https://user-images.githubusercontent.com/876340/191589620-70464ca1-6c95-440f-b27d-a1bc235ddf7a.mp4
